frame/staking/src/lib.rs

```diff
@@ -1922,14 +1859,6 @@ impl<T: Trait> Module<T> {
 					total,
 				};
 				<ErasStakers<T>>::insert(&current_era, &c, &exposure);
-
-				let mut exposure_clipped = exposure;
-				let clipped_max_len = T::MaxNominatorRewardedPerValidator::get() as usize;
-				if exposure_clipped.others.len() > clipped_max_len {
-					exposure_clipped.others.sort_unstable_by(|a, b| a.value.cmp(&b.value).reverse());
-					exposure_clipped.others.truncate(clipped_max_len);
-				}
-				<ErasStakersClipped<T>>::insert(&current_era, &c, exposure_clipped);
```

Comment:
There is a dilemma here:

Do we want to keep a copy of the stakers information for every staker for a bunch of eras, or do we want to have it so that some validators invoke reading of a vec which is large.

I think removing this redundant storage item is better, but could be wrong.